### PR TITLE
Update the list of resources

### PR DESCRIPTION
**Pull request by arybczak**

### CODE REVIEW

**Comment by phadej** [comment]
Should these talks have a year attached (and maybe sorted either chronologically, or in reverse).

**Comment by arybczak** [comment]
Sorted :slightly_smiling_face:

File: optics/src/Optics.hs [hunk 1]
```diff
@@ -734,17 +734,25 @@ import Data.Either.Optics                    as P
 
 -- $otherresources
 --
--- * <https://skillsmatter.com/skillscasts/10692-through-a-glass-abstractly-lenses-and-the-power-of-abstraction Through a Glass, Abstractly: Lenses and the Power of Abstraction> a talk on the principles behind this library with <https://github.com/well-typed/optics/raw/master/Talk.pdf accompanying slides> by Adam Gundry (but note that the design details of @optics@ have changed substantially since this talk was given)
+-- Talks:
```
Comment (phadej) [inline optics/src/Optics.hs:737]: should these be heading of some level, i.e.

```haskell
-- == Talks
```

https://hackage.haskell.org/package/haddock

Comment (arybczak) [reply optics/src/Optics.hs]: Done.

**Review by phadej — APPROVED** [review]

**Pull request merged**